app/controllers/Application.java

```diff
@@ -2170,7 +2173,7 @@ private static String getOptimizationAlgo(String tuningAlgorithmName) {
   }
 
   private static String reasonForDisablingTuning(TuningJobDefinition tuningJobDefinition) {
-    if (tuningJobDefinition.tuningEnabled) {
+    if (!tuningJobDefinition.tuningEnabled) {
```

Comment:
Why there is not equal ? earlier was equal to (was that a bug earlier)

Reply:
Yes it was a wrong condition used.